charts/node/values.yaml

```diff
@@ -57,9 +57,12 @@ node:
   #chainPath: ""
   #chainDataKubernetesVolumeSnapshot: ""
   #chainDataGcsBucketUrl: ""
+  enableStartupProbe: true
+  enableReadinessProbe: true
   flags:
     - "--prometheus-external"
     - "--rpc-external"
+    - "--ws-external"
     - "--rpc-cors"
```

Comment:
if you want to make this part more clear you could just use "--rpc-cors=all" on a single line.

Reply:
Thanks I changed it to what you suggested.